assets/misc/meilisearch-collection-postman.json

```diff
@@ -305,7 +305,7 @@
 						"method": "GET",
 						"header": [],
 						"url": {
-							"raw": "{{url}}/indexes/{{indexUID}}/search?q=prinec",
+							"raw": "{{url}}/indexes/{{indexUID}}/search?q=prinec&showRankingScore=true",
```

Comment:
`showRankingScore` should not be checked

<img width="1056" alt="Capture d’écran 2023-07-24 à 13 51 42" src="https://github.com/meilisearch/documentation/assets/20380692/f8c74b00-cfa4-43d3-9e43-0c1397477185">